make_tools.mk

```diff
@@ -1,12 +1,24 @@
+# This controls building executables in the `tools` folder.
+# Can be invoked through the `Makefile` or standalone.
 
 MAKEFLAGS += --no-print-directory
 
 # Inclusive list. If you don't want a tool to be built, don't add it here.
-TOOLDIRS := tools/aif2pcm tools/bin2c tools/gbafix tools/gbagfx tools/jsonproc tools/mapjson tools/mid2agb tools/preproc tools/ramscrgen tools/rsfont tools/scaninc
+TOOLS_DIR := tools
+TOOL_NAMES := aif2pcm bin2c gbafix gbagfx jsonproc mapjson mid2agb preproc ramscrgen rsfont scaninc
 
-.PHONY: all $(TOOLDIRS)
+TOOLDIRS := $(TOOL_NAMES:%=$(TOOLS_DIR)/%)
 
-all: $(TOOLDIRS)
+.PHONY: tools check-tools clean-tools $(TOOLDIRS) $(CHECKTOOLDIRS)
+
+tools: $(TOOLDIRS)
+check-tools: $(CHECKTOOLDIRS)
 
 $(TOOLDIRS):
 	@$(MAKE) -C $@
+
+$(CHECKTOOLDIRS):
+	@$(MAKE) -C $@
```

Comment:
Was this left behind from testing?

Reply:
seemingly so - don't actually remember. Removed